testmock/image.go

```diff
@@ -5,7 +5,7 @@
 package testmock
```

Comment:
It's a little weird that we still have this since the interface is no longer in this repo. @djoyahoy Is going to eliminate it by moving to the fake image instead of mock image. It may or may not be part of this PR.

Reply:
I suppose we could also duplicate the interface here on the consumer side (it will still be needed in imgutil as well though). Thoughts?